proto/sbom/sbom.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+option go_package = "github.com/DataDog/agent-payload/v5/sbom";
+
+package datadog.sbom;
+
+import "proto/deps/google/type/date.proto";
+import "proto/deps/github.com/CycloneDX/specification/schema/bom-1.4.proto";
+
+// SBOMPayload represents the main SBOM payload
+message SBOMPayload {
+  int32               version  = 1;
+  string              host     = 2;
+  string              source   = 3; // use to know the source of the message: agent, other
+  repeated SBOMEntity entities = 4;
+}
+
+message SBOMEntity {
+  SBOMSourceType   type        = 1;
+  string           id          = 2; // Unique identifier to be able to correlated and "deduplicate" SBOM
+  google.type.Date generatedAt = 3; // the datetime of the SBOM generation
```

Comment:
Do we need to use a Date instead of the standard `google/protobuf/timestamp.proto`?
It is not available in our side and I think the ts is more interesting than the date.

Reply:
I’ve just changed the type of this field to use `google.protobuf.Timestamp` instead.